tests/codebuild/run_all_sample_test.sh

```diff
@@ -65,3 +64,43 @@ if [ "$(kubectl smlogs batchtransformjob xgboost-batch | wc -l)" -lt "1" ]; then
     echo "smlogs batchtransformjob did not produce any output."
     exit 1
 fi
+
+# Clean up resources before re-using metadata names
+delete_all_tests
+
+verify_delete TrainingJob testfiles/xgboost-mnist-trainingjob.yaml
+
+training_job_name=$(kubectl get trainingjob xgboost-mnist -o json | jq -r ".status.sageMakerTrainingJobName")
+job_status=`aws sagemaker describe-training-job --training-job-name ${training_job_name} --region us-west-2 | jq -r ".TrainingJobStatus"`
```

Comment:
Wrapping in quotations will then require escaping the inner double quotations for variable names. I'm not sure how bash handles variable injection when escaping double quotations.

Reply:
I think you should try it out and see what happens :) Bash interprets double quotes differently than what we come to expect from other languages. In a nutshell, quoting expansions tells bash to treat it as a single term instead of splitting it on whitespace and processing special characters.

https://unix.stackexchange.com/a/131767/253226